master/internal/command/authz_iface.go

```diff
@@ -12,9 +12,19 @@ type NSCAuthZ interface {
 	// GET /api/v1/NSCs/:nsc_id
 	// GET /tasks
 	CanGetNSC(
-		ctx context.Context, curUser model.User, ownerID model.UserID, workspaceID model.AccessScopeID,
+		ctx context.Context, curUser model.User, workspaceID model.AccessScopeID,
 	) (canGetNsc bool, serverError error)
 
+	// GET /api/v1/NSCs
+	FilterNSCWorkspaces( // DELETEME.
```

Comment:
I think we just remove this and use AccessibleScopes

Reply:
yes I just left it in so it's easier to see and to comment on for you :)